_posts/2021-07-30-iceberg-concurrency-snapshots-spec.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: post
+title:  "Trino on ice III: Iceberg concurrency model, snapshots, and the Iceberg spec"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+canonical_url: www.starburst.io/blog/trino-on-ice-iii-iceberg-concurrency-model-snapshots-and-the-iceberg-spec/
+---
+
+<p align="center">
+ <img align="center" width="100%" height="100%" src="/assets/blog/trino-on-ice/trino-iceberg.png"/>
+</p>
+
+Welcome to the Trino on ice series, covering the details around how the Iceberg
+table format works with the Trino query engine. The examples build on each
+previous blog, so it's recommended to read the blogs sequentially and reference
+them as needed later. Here are links to the posts in this series:
+
+* [Trino on ice I: A gentle introduction To Iceberg]({% post_url 2021-05-03-a-gentle-introduction-to-iceberg %})
+* [Trino on ice II: In-place table evolution and cloud compatibility with Iceberg]({% post_url 2021-07-12-in-place-table-evolution-and-cloud-compatibility-with-iceberg %})
+* [Trino on ice III: Iceberg concurrency model, snapshots, and the Iceberg spec]({% post_url 2021-07-30-iceberg-concurrency-snapshots-spec %})
+* [Trino on ice IV: Deep dive into Iceberg internals]({% post_url 2021-08-12-deep-dive-into-iceberg-internals %})
+
+In the last two blog posts, we’ve covered a lot of cool feature improvements of
+Iceberg over the Hive model. I recommend you take a look at those if you haven’t
+yet. We introduced concepts and issues that table formats address. The next
+blogs will dive into some technical details to gain a better understanding
```

Comment:
blogs dive into

Reply:
also .. should that sentence move down a bit?

Reply:
You know...I tease it already at the end...I'm gonna remove it from there